app/assets/javascripts/covid19/src/webapp/components/structures-table/cells/PdbCell.tsx

```diff
@@ -14,9 +22,62 @@ export const PdbCell: React.FC<CellProps> = React.memo(props => {
 
 const PdbCell2: React.FC<{ structure: Structure; pdb: Pdb }> = React.memo(props => {
     const { pdb, structure } = props;
+    const [open, setOpen] = React.useState(false);
     const pdbValidations = structure.validations.pdb;
+    const t = React.useMemo(getTranslations, []);
 
-    const tooltip = (
+    const getValidation = React.useCallback((pdbValidation: PdbValidation) => {
+        switch (pdbValidation?.type) {
+            case "pdbRedo":
+                return (
+                    <GroupBadges key="pdb-redo">
```

Comment:
Refactor. A useCallback should not return JSX. JSX only in the final return of the function component.